utils/install.py

```diff
@@ -342,7 +342,7 @@ def compare(self, version2, separator=". |-", ignorecase=True):
     WASMEDGE_TENSORFLOW_PLUGIN,
     WASMEDGE_TENSORFLOW_LITE_PLUGIN,
     WASMEDGE_IMAGE_PLUGIN,
-    WASM_BPF
+    WASM_BPF,
```

Comment:
I would like to know if we can have a Python linter in the test-installer.yml workflow.

Reply:
Done, added it